.github/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policiy
+
+The TypeScript ESLint team and community take all security vulnerabilities seriously.
+If you have a vulnerability or other security issues to disclose:
+
+- Thank you very much, please do!
+- Please send them to us by emailing `me@joshuakgoldberg.com`
```

Comment:
@JamesHenry is looking into if we can create a generic `@typescript-eslint.io` email address for the project rather than using a personal one.

Reply:
Pending the forwarding email:

```suggestion
- Please send them to us by emailing `typescripteslint@gmail.com`
```